news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: How advances in technology affected the osu! community
+date: 2023-12-12 12:30:00 +0000
+---
+
+Let's walk back through the past 15 years of osu!, and how various new technologies affected the game and the community.
+
+![]()
+
+Every now and then there's a new piece of tech that's slated to change the way games are played, whether it'd be a new input device, a new way to squeeze more FPS out of your PC, or a whole new way to experience games.
+
+While the osu! community has managed to stay unswayed as things such as virtual reality and augmented reality, cloud gaming & AI advancements cropped up due to them being inapplicable to osu!, there were still some things that changed the way osu! is played, or at least changed the view of the most optimal way to play it.
+
+It's time to dive in and see how technology can impact the community, what the results of widespread adoption are, or could have been.
+
+## Graphics tablets (2007)
+
+osu! originally started out as an *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)* clone for PC, which was a game that was exclusive to the Nintendo DS, a console that came, and could be used with the included stylus pen. When [peppy](https://osu.ppy.sh/users/2) released osu!, it was only natural for players who already had a graphics tablet to try out osu! with it, and since the get-go it was considered a commodity input device that would more closely replicate the original Nintendo DS experience.
+
+Playing with a tablet is a completely different ball game compared to the mouse, but rather than being a direct upgrade by eliminating mouse drift with its absolute positioning, it brings up new challenges and questions regarding its ideal usage. Ideal pen grip, tablet area, dragging or hovering play styles have all been detrimental to increasing the learning curve, which is why some players choose to remain on the much more familiar and intuitive mouse.
+
+Over the years many top players and regular players started switching over from mouse to tablet in order to avoid issues associated with mice, however there have always been top players with both input devices, and they are both considered viable for top-level play, each with its own upsides and downsides.
```

Comment:
```suggestion
Over the years, many top players and regular players started switching over from mouse to tablet in order to avoid issues associated with mice, however there have always been top players with both input devices, and they are both considered viable for top-level play, each with its own upsides and downsides.
```

you mean like one player playing both tablet and mouse or that top players always included some tablet and some mouse players, the sentence would suggest former i think? :thinking:

Reply:
I meant to say that once tablets were easier to obtain that a lot of top players swapped to it to trade the downsides of the mouse with the downsides of the tablet, but top-level play is achievable with both still

Reply:
the original sentence sucked let me know if this is better now
https://github.com/ppy/osu-wiki/pull/10659/commits/2f9b7b1f66feda3316da94dd668da811023eb2f8

Reply:
sounds way better indeed, posted some minor wording suggestion at https://github.com/ppy/osu-wiki/pull/10659#discussion_r1429220780